src/app/shell/personal-cabinet/provider/create-competition/create-competition.component.spec.ts

```
@@ -0,0 +1,73 @@
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CreateCompetitionComponent);
+    component = fixture.componentInstance;
+    component.RequiredFormGroup = new FormGroup({});
+    component.ContactsFormArray = new FormArray([]);
+    component.DescriptionFormGroup = new FormGroup({});
+    component.JudgeFormArray = new FormArray([]);
+
+    fixture.detectChanges();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)